### ISSUE TITLE
Add method for tree testing

### ISSUE THREAD

**Issue by pglevy** [Bixal/methods#16]
Create a new method that explains the tree testing method.

## Acceptance criteria
- [ ] Includes all user-facing content:
  - [ ] title
  - [ ] what
  - [ ] why
  - [ ] how
  - [ ] phase
  - [ ] time required
  - [ ] resources
  - [ ] considerations
- [ ] Includes additional front matter content as needed.
- [ ] Considers input from members of the UX team.
- [ ] Includes consideration for specific tools and templates.
- [ ] Includes consideration for how to conduct with widely available tools.
- [ ] Follow [plain language guidelines](https://accessibility.digital.gov/content-design/plain-language/), including checking against the [plainlanguage.gov list for web](https://www.plainlanguage.gov/resources/checklists/web-checklist/).

## Resources to consider
- [Optimal Workshop](https://www.optimalworkshop.com/learn/101s/tree-testing/) 
- [NNG](https://www.nngroup.com/articles/tree-testing/)

**Comment by pglevy** [comment]
Got something [started here](https://github.com/Bixal/methods/blob/tree-testing/_methods/tree-testing.md). Still need to fill in detail on "how," but most of the other details in place (example, resources, etc). Started from the Usability Testing method.

**Comment by sofya-UX** [comment]
Looking good @pglevy. Thank you for getting this off the ground.
While reading the content, I notices the footer with the PRA boilerplate needs tweaking. I found this play on PRA approval, and thought that it was the clearest explanation of this that I have ever read yet: https://www.usda.gov/digital-strategy/research/plays
Given that the statistically significant sample for tree test method is between 30-50 people according to OW (and some other resources recommend 50 as a minimum), we need to adjust the PRA language saying that this study is subject to PRA approval (I suspect it will be true for the Card Sorting method, too)

**Comment by pglevy** [comment]
any progress on this, @sofya-UX ?

**Comment by sofya-UX** [comment]
@pglevy taking a pass at it today. Is it ok for me to just edit the page and commit changes?

**Comment by pglevy** [comment]
yes, thanks

**Comment by sofya-UX** [comment]
The Tree Test method draft is in a good place for your review and feedback @Bixal/methods here: https://github.com/Bixal/methods/blob/tree-testing/_methods/tree-testing.md

**Comment by pglevy** [comment]
I've done an editorial pass of the content. Re: input from others, we'll have to leave more time in the cycle for that in the future. Having the three of us look is fine for now. Re: time required, I used the guidance from the Usability testing method for reference. I've started a [discussion here](https://github.com/Bixal/methods/discussions/72) where we can work through the questions and issues around this content.

Marking as done for now, but I'll merge in with the other stuff for release next week.
